intranet/static/js/login.js

```diff
@@ -57,4 +69,4 @@ $(document).ready(function() {
         }
     });
 
-});
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2743148458/issues/source?bid=3176160&fileBranchId=3355598#l72)